services/quests/docs/qdl_guide.md

```diff
@@ -310,21 +336,85 @@ To do this, you simply annotate the combat on win outcome. Below, you can see wh
 
 Note: In cases where adventurers won't heal at the end of combat, it's nice to warn them with an instruction node before combat starts so that they can make better choices - for example: `> You're on the run! You won't heal after this encounter.`
 
+### Decision cards
 
-### Roleplaying Cards
+Decisions allow you to add a more structure element of action to your quest. These can happen both in combat and during roleplay - in fact, they can happen automatically in combat if The Future expansion is enabled!
 
-Everything except for combat happens in roleplay cards, including showing text and choices to adventurers. The basic syntax for a roleplaying card is:
+A decision plays out similarly to a round of combat - there's a timed phase where users have a limited amount of time to select the best action to take given some scenario you present.
 
-```
-_Card Title_
+Decisions can be used without The Future, but they really shine when users have Skill cards that they can use to affect their success or failure when attempting a skill check. Try to use these only in quests where you mark The Future as a requred expansion.
```

Comment:
Can they, though? I feel like we would need to make a separate instruction set in the app for if there's a skill check without Skill cards...

Reply:
Fix'd.